db/migrate/20190522114510_migrate_subscriber_lists_to_and_joined_facet_subscriber_lists.rb

```diff
@@ -0,0 +1,5 @@
+class MigrateSubscriberListsToAndJoinedFacetSubscriberLists < ActiveRecord::Migration[5.2]
+  def change
+    SubscriberList.where(type: nil).update_all(type: 'AndJoinedFacetSubscriberList')
```

Comment:
Is it really necessary to do this? Isn't it less risky to just create a new Subscriber List for "ORs" rather than migrating all of the old ones as well?